<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWIndow</class>
 <widget class="QMainWindow" name="MainWIndow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>550</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>720</width>
    <height>550</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>720</width>
    <height>550</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #1e1f1e;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="logo">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>30</y>
      <width>199</width>
      <height>50</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="pixmap">
     <pixmap resource="resources/rsrcs.qrc">:/jpeg/logo/rsdx.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="wlcText">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>110</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #353535;
border-radius:10px;
color: white;
padding: 10px;</string>
    </property>
    <property name="text">
     <string>Registration for new users</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="regBtn">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>380</y>
      <width>221</width>
      <height>71</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">* {border: 1px solid rgb(53, 132, 228);
		color:white;
		background: rgb(53, 132, 228);
		font-size: 20px;
		border-radius:10px;
		padding:20px;}
       *:hover{ background:none;border: 1px solid rgb(53, 132, 228); color: white; }</string>
    </property>
    <property name="text">
     <string>SIGN UP</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="uname">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>180</y>
      <width>431</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #fff;
padding: 10px;
color: rgb(94, 92, 100);
border: 1px solid rgb(94, 92, 100);
border-radius: 5px;</string>
    </property>
    <property name="placeholderText">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="passwd">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>240</y>
      <width>431</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #fff;
padding: 10px;
color: rgb(94, 92, 100);
border: 1px solid rgb(94, 92, 100);
border-radius: 5px;</string>
    </property>
    <property name="placeholderText">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="passwd_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>300</y>
      <width>431</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #fff;
padding: 10px;
color: rgb(94, 92, 100);
border: 1px solid rgb(94, 92, 100);
border-radius: 5px;</string>
    </property>
    <property name="placeholderText">
     <string>Retype Password</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loginBtn">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>380</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">* {border: 1px solid rgb(51, 209, 122);
		color:white;
		background: rgb(51, 209, 122);
		font-size: 15px;
		border-radius:10px;
		padding:10px;}
       *:hover{ background:none;border: 1px solid rgb(51, 209, 122); color: white; }</string>
    </property>
    <property name="text">
     <string>ALREADY HAVE AN
ACCOUNT?</string>
    </property>
   </widget>
   <widget class="QLabel" name="errorMsg">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>110</y>
      <width>391</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(237, 51, 59);
border-radius:10px;
color: white;
padding: 10px;</string>
    </property>
    <property name="text">
     <string>user already exists or passwords didn't match</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="nav_homeBtn">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>480</y>
      <width>90</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">*{background: none;
color: white;
border: none;
border: 1px solid transparent}
*:hover{ border-bottom: 1px solid white}</string>
    </property>
    <property name="text">
     <string>Home</string>
    </property>
   </widget>
   <zorder>wlcText</zorder>
   <zorder>regBtn</zorder>
   <zorder>logo</zorder>
   <zorder>uname</zorder>
   <zorder>passwd</zorder>
   <zorder>passwd_2</zorder>
   <zorder>loginBtn</zorder>
   <zorder>errorMsg</zorder>
   <zorder>nav_homeBtn</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resources/rsrcs.qrc"/>
 </resources>
 <connections/>
</ui>
